lib/private/L10N/Factory.php

```diff
@@ -58,6 +58,12 @@ class Factory implements IFactory {
 		'pt_BR', 'pt_PT', 'da', 'fi_FI', 'nb_NO', 'sv', 'tr', 'zh_CN', 'ko'
 	];
 
+	const RTL_LANGUAGES = [
```

Comment:
Maybe make the visibility explicit?

```suggestion
	public const RTL_LANGUAGES = [
```